source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -126,172 +130,137 @@ used at the data preparation and data processing stages for the following:
+  sub-type or a property of table metadata.
 
-- Iterative construction of a :txtref:`table` from another :txtref:`<table>s` or
-  a different in-memory :txtref:`dataset` representations.
-
-- Construction of a :txtref:`table` from different entities that hold blocks of
-  data, such as arrays, pointers to the memory, external entities.
-
-- Changing dataset values. Since :txtref:`table` is an
-  :capterm:`immutable <immutability>` dataset, a builder provides the ability to
-  change the values in a dataset under construction.
-
-- Encapsulating construction process of a :txtref:`table`. This is used to hide the
-  implementation details as they are irrelevant for users. This also allow to
-  select the most appropriate table implementation for each particular case.
-
-- Providing additional information on how to create a :txtref:`table` inside an
-  algorithm for :txtref:`<result>s`. This information includes metadata, memory
-  allocators that shall be used, or even a particular table implementation.
-
-For details, see :txtref:`table-builders` section.
+  For example, :capterm:`data layout` and :capterm:`data format` are properties
+  of table objects since they affect the structure of a table, its contract, and
+  behavior.
 
 .. _accessor:
 
 Accessor
 --------
 
-Accessor is a concept that defines a single way to get the data from an
-in-memory numerical :txtref:`dataset`. It allows:
+Accessor is a concept that defines a single way to extract the data from a
+:txtref:`table`. It allows to:
 
-- To have unified access to the data from various sets of different objects,
-  such as :txtref:`<table>s` or :txtref:`<table-builder>s`,
-  without exposing their implementation details.
+- Have unified access to the data from :txtref:`table` objects of different
+  types, without exposing their implementation details.
 
-- To convert a variety of numeric :capterm:`data formats <data format>` into a
-  smaller set of formats.
+- Provide a :capterm:`flat <flat data>` view on the data blocks of a
+  :txtref:`table` for better data locality. For example, some accessor
+  implementation returns column values as a contiguous array, while the original
+  table stored row-by-row (there are strides between values of a single
+  column).
 
-- To provide a :capterm:`flat <flat data>` view on the data blocks of a
-  :txtref:`dataset` for better a data locality. For example, some accessor
-  implementation returns :capterm:`feature` values as a contiguous array, while
-  the original dataset stored row-by-row (there are strides between values of a
-  single feature).
-
-- To acquire data in a desired :capterm:`data format` for which
+- Acquire data in a desired :capterm:`data format` for which
   a specific set of operations is defined.
 
-- To have read-only, read-write and write-only access to the data. Accessor
+- Have read-only, read-write and write-only access to the data. Accessor
   implementations are not required to have read-write and write-only access
```

Comment:
.. are not required to support read-write and write-only access for ..entities such as...